test/const.test.js

```diff
@@ -14,14 +14,13 @@ test('schema with const string', (t) => {
     }
   }
 
-  const validate = validator(schema)
+  const input = { foo: 'bar' }
+  const validate = ajv.compile(schema)
   const stringify = build(schema)
-  const output = stringify({
-    foo: 'bar'
-  })
 
-  t.equal(output, '{"foo":"bar"}')
-  t.ok(validate(JSON.parse(output)), 'valid schema')
+  t.equal(stringify(input), '{"foo":"bar"}')
+  t.ok(validate((input)))
```

Comment:
we are validating our test code here
```suggestion
  t.ok(validate(JSON.parse(output)))
```